### ISSUE TITLE
Duration does not apply.

### ISSUE THREAD

**Issue by SJFAFA** [infotrex/bulk-upload-to-opensea#102]
The signing problem was solved with the draft script you sent.
but..
After checking the duration of 180 days, press the start button
It continues to be created with a sales duration of 7 days.
I want 6 months

**Comment by abhiram-ms** [comment]
yes for me too from today even if the duration is checked 180 days its only showing 7 days in open sea it was working smoothly but now its not working what happened suddenly
